scripts/fix_db/verify_read_model.py

```diff
@@ -0,0 +1,79 @@
+from sqlalchemy import select
```

Comment:
The `scripts/fix_db` folder will be deprecated, when we implement of https://github.com/fractal-analytics-platform/fractal-server/issues/1094#issuecomment-1893931516 - let's move this script somewhere else.

I propose `scripts/validate_db_data_with_read_schemas.py`